src/jti_background.js

```diff
@@ -161,11 +174,24 @@ function fetchDefaultTemplates (callback) {
     });
 }
 
+function getData (callback) {
+    chrome.storage.sync.get(StorageID, function (templates) {
+        if (templates[StorageID]) {
+            var templateJSON = templates[StorageID];
+            templateJSON.templates = templateDataToJSON(templateJSON.templates);
+            templateJSON.options.domains = domainDataToJSON(templateJSON.options.domains);
+            templateJSON.options.inputIDs = inputIDDataToJSON(templateJSON.options.inputIDs);
+            callback(true, '', templateJSON);
+        } else {
+            callback(false, 'No data is currently loaded');
+        }
+    });
+}
+
 function setDefaultTemplates (callback) {
     fetchDefaultTemplates(function (status, message, data) {
         if (status) {
-            data.templates = JSONtoTemplateData(data.templates);
-            data.options.domains = JSONtoDomainData(data.options.domains);
+            data = JSONtoData(data);
             saveTemplates(data, callback);
```

Comment:
We can just combine these two lines
`saveTemplates(JSONtoData(data), callback);`

Reply:
Sounds good